### PR TITLE
Added Shortcuts + some improvements

### PR DESCRIPTION
**Pull request by LostInDarkMath**

### CODE REVIEW

File: report.tex [hunk 1]
```diff
@@ -15,6 +24,21 @@
 \bibliographystyle{plain}	% Literaturverzeichnisformat = standard
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%OTHER OPTIONS
+\setlength{\parindent}{0px} %dieser Befehl verhindert das Einrücken eines Absatzes bei einer leeren Codezeile.
```
Comment (kaewik) [inline report.tex]: I would prefer the standard behavior of latex.  Should not be set in template.

File: report.tex [hunk 2]
```diff
@@ -1,8 +1,17 @@
 \documentclass{scrreprt}	%KOMA-Script Klasse für report; 
 
 % HEADER
-\usepackage[english, german]{babel}	% Paket für Sprachselektion, in diesem Fall für deutsches Datum etc
+\usepackage[english, ngerman]{babel}	% Paket für Sprachselektion, in diesem Fall für deutsches Datum etc
 \usepackage[utf8]{inputenc}	% Paket für Umlaute; verwende utf8 Kodierung in TexWorks 
+\usepackage[T1]{fontenc} %ö,ü,ä werden richtig kodiert
```
Comment (kaewik) [inline report.tex]: Why is this needed? The package above should do the trick.

File: report.tex [hunk 3]
```diff
@@ -15,6 +24,21 @@
...
+%OTHER OPTIONS
+\setlength{\parindent}{0px} %dieser Befehl verhindert das Einrücken eines Absatzes bei einer leeren Codezeile. 
+
+%SHORTCUTS
+\newcommand{\R}{\mathbb{R}} %reelle Zahlen
+\newcommand{\N}{\mathbb{N}} %natürliche Zahlen
+\newcommand{\Z}{\mathbb{Z}} %ganze Zahlen
+\newcommand{\C}{\mathbb{C}} %komplexe Zahlen
+
+\newcommand{\stackeq}[1]{\mathrel{\stackrel{\makebox[0pt]{\mbox{\normalfont\tiny #1}}}{=}}} %das ist ein Gleichheitszeichen mit Text darüber, Beispiel: $a\stackeq{Def} b$
```
Comment (kaewik) [inline report.tex]: As the newcommand below defines a more generic approach, this can be omitted.

Comment (kaewik) [reply report.tex]: Or at least use the command below to define this one. Code duplication!

File: report.tex [hunk 4]
```diff
@@ -15,6 +24,21 @@
...
+\newcommand{\stacksymbol}[2]{\mathrel{\stackrel{\makebox[0pt]{\mbox{\normalfont\tiny #1}}}{#2}}} %das ist ein beliebiges Zeichen mit Text darüber, z. B. $a\stackrel{Def}{\Rightarrow} b$
+\newcommand\tab[1][1cm]{\hspace*{#1}} %praktischer Tabulator
```
Comment (kaewik) [inline report.tex]: What does this mean?

**Review by kaewik — CHANGES_REQUESTED** [review]
Nice work. Some really nice things are added. Please be so kind and outsource the new commands and short cuts in an extra package. You may called it commands.sty or something like that. The same goes for the packages.

File: report.tex (hunk 2 above)
Comment (LostInDarkMath) [reply report.tex]: http://texwelt.de/wissen/fragen/5537/was-macht-eigentlich-usepackaget1fontenc

I think the main difference is the following: For example if you search in the pdf for  "können" you won't find it without this package.

File: report.tex (hunk 1 above)
Comment (LostInDarkMath) [reply report.tex]: That's a matter of taste. Maybe a good compromie would be, that it is commented out per default.

File: report.tex (hunk 3 above)
Comment (LostInDarkMath) [reply report.tex]: Right! I'll fix this.

File: report.tex (hunk 2 above)
Comment (kaewik) [reply report.tex]: Do we then still need the package above?

File: report.tex (hunk 1 above)
Comment (kaewik) [reply report.tex]: Good idea!

File: report.tex (hunk 2 above)
Comment (LostInDarkMath) [reply report.tex]: Absolutly! Just try it out :)

File: report.tex (hunk 4 above)
Comment (LostInDarkMath) [reply report.tex]: Do you mean the \tab-Command? 

This is just a selfmade tabulator for formatting source code etc. in LaTeX.

File: report.tex [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
 \usepackage[utf8]{inputenc}	% Paket für Umlaute; verwende utf8 Kodierung in TexWorks 
+\usepackage[T1]{fontenc} %ö,ü,ä werden richtig kodiert
+
+\usepackage{amsmath} %wichtig für align-Umgebung
+\usepackage{amssymb} %wichtig für \mathbb{} usw.
+\usepackage{amsthm} %damit kann man eigene Theorem-Umgebungen definieren, proof-Umgebungen, etc.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% THEOREM-ENVIRONMENTS
+\newtheorem{satz}{Satz} %wird noch verbessert hier
```
Comment (kaewik) [inline report.tex]: No German defaults please. The template shall also be template for English documents.

File: report.tex (hunk 2 above)
Comment (kaewik) [reply report.tex]: I tested it we don't need that. The find functionality works with standard utf-8

**Comment by LostInDarkMath** [comment]
PUSH!

**Review by kaewik — APPROVED** [review]
Nice 😁
